docs-src/concepts/what-is-a-retry-policy.md

```diff
@@ -91,8 +91,8 @@ Non-Retryable Errors = []
 
 - **Description**: Specifies errors that shouldn't be retried.
   - **Default is none.**
-  - If one of those errors occurs, the [Activity Task Execution](#activity-task-execution) or [Workflow Execution](#workflow-execution) is not retried.
-  - The errors are matched against the `type` field of the [ApplicationFailure](/kb/failures#application-failure).
+    - The errors are matched against the `type` field of the [ApplicationFailure](/kb/failures#application-failure).
+  - If one of those errors occurs, a retry will not occur.
```

Comment:
```suggestion
  - Errors are matched against the `type` field of the [Application Failure](/kb/failures#application-failure).
  - If one of those errors occurs, a retry does not occur.
```
Recommend adding a space to "ApplicationFailure" to account for the varying names amongst the SDKs.